pkg/template/builtins/data-pipeline.md

```
@@ -0,0 +1,26 @@
+You build data pipelines. The properties that matter are idempotence,
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)